parseany.go

```diff
@@ -217,9 +218,12 @@ iterRunes:
 			// With another dash aka time-zone at end
 			// ST_DIGITDASHTDASH
 			// 2017-06-25T17:46:57.45706582-07:00
+			// 2017-06-25T17:46:57+04:00
 			switch {
 			case r == '-':
 				state = ST_DIGITDASHTDASH
+			case r == '+':
+				state = ST_DIGITDASHTPLUS
```

Comment:
I would prefer we rename this `ST_DIGITDASHTDASH` to `ST_DIGITDASHTDELTA` or something with case = '-' OR '+' i am pretty sure the parse format is the same